legacy/library/HelloSignLegacy/Client.php

```diff
@@ -640,7 +640,7 @@ public function createUnclaimedDraftEmbeddedWithTemplate(EmbeddedSignatureReques
     {
         $url = static::UNCLAIMED_DRAFT_CREATE_EMBEDDED_WITH_TEMPLATE_PATH;
 
-        $response = $this->rest->post($url, $request->toParams());
+        $response = $this->rest->post($url, $request->toUnclaimedDraftParams());
```

Comment:
here i'm adding a new function to target the specific endpoint `unclaimed_draft/create_embedded_with_template`, since the other  endpoint that uses `EmbeddedSignatureRequest`, `signature_request/create_embedded and `signature_request/create_embedded_with_template` does not support parameters `show_preview` and 'preview_only'.